databases/week1/DB_Week1.sql

```diff
@@ -0,0 +1,38 @@
+-- 1
+SELECT COUNT(id) FROM task;
+
+-- 2
+SELECT COUNT(id) FROM task
+WHERE due_date IS NULL;
+
+-- 3
+SELECT task.title AS task_name, status.name AS status_name FROM task JOIN status ON task.status_id = status.id
+WHERE status.name = 'Done';
+
+-- 4
+SELECT task.title AS task_name, status.name AS status_name FROM task JOIN status ON task.status_id = status.id
+WHERE status.name <> 'Done';
+
+-- 5
+SELECT * FROM task ORDER BY created DESC;
+
+-- 6
+SELECT * FROM task ORDER BY created DESC LIMIT 1;
+
+-- 7
+SELECT task.title, task.due_date FROM task
```

Comment:
To make sure the result is right, it would the good to select also the task.description